components/cam/src/physics/rrtmg/radiation.F90

```diff
@@ -2144,7 +2196,11 @@ subroutine radiation_tend( state, ptend,pbuf, &
 
             ! Convert upward longwave flux units to CGS
             do i=1,ncol
-              lwupcgs(i) = cam_in%lwup(i)*1000._r8
+#ifdef MAML
+            lwupcgs(i) = cam_in%lwup(i,ii)*1000._r8
```

Comment:
L2200 and L2202 should be indented one more level.

Reply:
It's also worth noting that all indents should be done with spaces, so if these misalignments are due to tab characters then those should be changed.